templates/3_Pixie_Cluster_Cells.ipynb

```diff
@@ -297,7 +297,9 @@
    "cell_type": "markdown",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.        cluster_counts_size_norm = pd.read_dataframe(os.path.join(base_dir, cluster_counts_size_norm_name))
</pre><p> Same here - I think this should be <code>feather.read_dataframe</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/999/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>